OracleWebLogic/samples/12212-msiserver/README.md

```diff
@@ -0,0 +1,160 @@
+Example of docker images with WebLogic server in MSI Mode
+=========================================================
+This Dockerfile extends the Oracle WebLogic image by creating a domain that configures a managed
+server in MSI mode (or Managed Server Independence mode). In this mode, a managed server can run
+without the need of an admin server. Such a managed server is not driven by admin server for
+configuration or deployment changes. However, it can handle all configuration and deploymnets
+already in config.xml like any other managed server. For use cases where the managed server does
+not need to be updated for configuration or deployments, this image can be used by itself without
+running a admin server or node manager
+
+How to build and run the base image
+-----------------------------------
+First make sure you have built **oracle/weblogic:12.2.1.2-developer**. 
+
+Next, to build the base msi image, run:
+
+        $ docker build -t 12212-msiserver .
+
+Finally, to start the Managed Server in MSI mode, run:
+
+        $ docker run -d -p 8011:8011 12212-msiserver
+
+Connect to this container instance. You'll notice that the managed server is running from a domain
+located at /u01/msi-server. Under the servers directory, you'll notice a server name that seems
+randomly generated. It is of pattern ms[0-9]*. Say that your generated container id is cf579fd131fc
+and you find that the random managed server name is ms3. You can verify the server is in MSI mode
+by searching for a catalog message BEA-150018
+
+$ docker ps
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                    NAMES
+cf579fd131fc        12212-msiserver     "/u01/oracle/launc..."   2 minutes ago       Up 2 minutes        0.0.0.0:8011->8011/tcp   vigilant_volhard
+
+$ docker exec -it cf579fd131fc bash

```

Comment:
You should never need to exec `bash` into a container. If you need the user to know something, that log file should be sent to `stdout` at runtime so that the user can use `docker logs` (or any of the logging drivers) to get the output.